### DISCUSSION TITLE
Rejected from GitHub developer pack due to incorrect email address in database

### DISCUSSION THREAD

**Discussion by AtlasNavigator** [community/community#135705] [GitHub Education]
### Hi everyone,

Hello GitHub Education,

I am a student at Fresno City College, which operates within the State of California Community College District. When applying, I was unable to proceed with the application because it required an @fresnocitycollege.edu email address, which is only provided to staff. Students only get @my.scccd.edu emails. How can I proceed with the application. I have already been denied when I entered my institution's information manually and uploaded an image of my student ID.

**Comment by davevad93** [comment]
Hi @AtlasNavigator ,

I don't know exactly how you could proceed with your request since the required email domain for your school is `@fresnocitycollege.edu`, however I can suggest you to contact Education support by opening a ticket [here](https://support.github.com/contact/education), select "`My selected school might have incorrect or incomplete information e.g. domains or campus location`" and explain the problem with your edu email.
